certified-connectors/DocuSignDemo/script.csx

```diff
@@ -1506,7 +1506,7 @@ public class Script : ScriptBase
     if ("GetRecipientFields".Equals(this.Context.OperationId, StringComparison.OrdinalIgnoreCase))
     {
       var uriBuilder = new UriBuilder(this.Context.Request.RequestUri);
-      uriBuilder.Path = uriBuilder.Path.Replace("/recipientsV2", "/recipients");
+      uriBuilder.Path = uriBuilder.Path.Replace("/recipientFields", "/recipients/");
```

Comment:
why do you need the extra '/' after `/recipients` ?

Reply:
That is not needed. I will remove it.